downstream/assemblies/hub/assembly-syncing-to-cloud-repo.adoc

```diff
@@ -1,7 +1,7 @@
 [id="assembly-creating-tokens-in-automation-hub"]
-= Configuring {HubName} remote repositories to sync content from Red Hat Certified and Ansible Galaxy collections
+= Configuring {HubNameMain} remote repositories to sync content from Red Hat Certified and {Galaxy} collections
```

Comment:
Not sure if for today, but this is a pretty long title (much more than 12 words). It would get cut off in a Google search. Could we use "Red Hat collections" and then just make sure the products are spell out in the short description below the title?